src/vuex/modules/data/actions.js

```diff
@@ -45,6 +46,7 @@ export const listIndexesAndCollections = (store) => {
             }
             if (index !== '%kuzzle') {
               addLocalRealtimeCollections(result, index)
+              result = dedupeRealtimeCollections(result)
```

Comment:
Hum, 2 point here: 
- `dedupeRealtimeCollection()` should be called after each call of `addLocalRealtimeCollections()` -> so you've missed a [call at line 87](https://github.com/kuzzleio/kuzzle-backoffice/blob/627a7ae802858e696d764d93203c52c385b5ea1a/src/vuex/modules/data/actions.js#L87)
- OR `dedupeRealtimeCollection()` logic should be added into `addLocalRealtimeCollections()`

What did you think ?